README.md

```diff
@@ -143,11 +151,13 @@ All ReportingApi methods have an optional _suite_ parameter.<br/>
 If you want to add a data to the suite, you must pass the suite name as the last parameter.
 
 ##### addAttributes
```

Comment:
Please update this section with a small code example and mention that users can now specify test attributes in test titles.